R2API/R2API.csproj

```diff
@@ -40,6 +40,10 @@
       <HintPath>libs\Facepunch.Steamworks.dll</HintPath>
       <Private>false</Private>
     </Reference>
+    <Reference Include="RoR2BepInExPack">
```

Comment:
Any future plans to publish this as a nuget?

Reply:
Yeah totally, i'll do it on a separate PR, its not urgent